sdk/samples/iot/README.md

```diff
@@ -155,22 +155,28 @@ To run the samples, ensure you have the following programs or tools installed on
 
 Samples use environment variables for a variety of purposes, including filepaths and connection parameters. Please keep in mind, **every time a new terminal is opened, the environment variables will have to be reset**. Setting a variable will take the following form:
 
-Linux:
+**Linux**:
```

Comment:
nit: Make it a `####` heading?

Reply:
I see.  Yes, if we want these to be bold, then according to the new linter they should use #### like "#### IoT Hub Certificate Samples" further below.

Reply:
I don't think the linter complains about using `**` for bolding. So, this was just a formatting suggestion (to use headings), not necessarily one to make the linter happy.

However, if you do use `### Linux` then you have to drop the colon, because that is when the linter will complain

![image](https://user-images.githubusercontent.com/6527137/89854731-847b7380-db49-11ea-94a5-a3857bd531fd.png)

So, if you want to keep the colon, leave it as `**Linux**:`